### PR TITLE
SystemConfigException fixed

### PR DESCRIPTION
**Pull request by gab-umich**
Fixed Issue https://github.com/yahoo/fili/issues/927#issue-459277455

new tests written and passed
logger in place

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: fili-system-config/pom.xml [hunk 1]
```diff
@@ -40,6 +40,11 @@
             <artifactId>spock-core</artifactId>
             <scope>compile</scope>
         </dependency>
+        <dependency>
```
Comment (gab-umich) [inline fili-system-config/pom.xml]: Not sure if this bunch is needed. 
Please double check.

File: fili-system-config/pom.xml [hunk 2]
```diff
@@ -40,6 +40,11 @@
...
+            <groupId>org.spockframework</groupId>
+            <artifactId>spock-core</artifactId>
+            <scope>compile</scope>
```
Comment (will-lauer) [inline fili-system-config/pom.xml]: This appears to be a mistake. At best, spock-core should be test scope

Comment (gab-umich) [reply fili-system-config/pom.xml]: Addressed. Thanks for pointing it out.

**Comment by gab-umich** [comment]
Just a reminder for you that I don't have write access to the yahoo/fili repo yet. 
Someone should merge this into master as long as this passes previous tests.

**Pull request merged**